### DISCUSSION TITLE
Add crypto donations

### DISCUSSION THREAD

**Discussion by manreo** [rxhanson/Rectangle#421] [General]
Hi,
I love your app, it would be great if you add the possibility to have crypo donations?
For me the best it bitcoin/ lightning (which you can add here - https://tippin.me/).
But also https://blog.coinpayments.net/news-features/cryptocurrency-donations is ok, as it supports many different coins (make sure to enable bitcoin and bitcoin lightning)

Thanks!

**Comment by Sehyn** [comment]
Follow up on this.

**Comment by rxhanson** [comment]
Thanks for looking to donate, and for the suggested methods.

I looked at this for a little while and went with coinpayments. The button is on https://rectangleapp.com.

I got scared away from tippin.me because of the disclaimer: 

> Tippin is still in beta, and you *could* lose funds. Be careful and cash out often.

I also looked at BTCPayServer, since a lot of people seem to recommend it, but I don't want to pay monthly hosting fees to "self" host it in case not many people want to donate crypto.

**Comment by manreo** [comment]
coinpayments is not a bad option, but I don't see it on the website..

**Reply by rxhanson** [reply]
I had it up there for about a month and didn't receive any donations, so I felt like it is better to just have the simplicity of the two Paypal buttons. I'll consider putting it back up if this discussion gets more upvotes.
